core/src/main/java/org/opensearch/sql/analysis/JsonSupportAnalyzer.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.analysis;
+
+import org.apache.commons.lang3.StringUtils;
+import org.opensearch.sql.ast.AbstractNodeVisitor;
+import org.opensearch.sql.ast.Node;
+import org.opensearch.sql.ast.expression.Alias;
+import org.opensearch.sql.ast.expression.Cast;
+import org.opensearch.sql.ast.expression.Function;
+import org.opensearch.sql.ast.expression.Literal;
+import org.opensearch.sql.ast.tree.Aggregation;
+import org.opensearch.sql.ast.tree.Project;
+
+public class JsonSupportAnalyzer extends AbstractNodeVisitor<Boolean, JsonSupportAnalysisContext> {
```

Comment:
Context is never used actually, so you can do
```suggestion
public class JsonSupportAnalyzer extends AbstractNodeVisitor<Boolean, Object> {
```
And pass `null`.

Reply:
Addressed in cc7893362586496172ba67e4c614ca78faa515f2